.github/CONTRIBUTING.md

```diff
@@ -22,8 +22,11 @@ Once you have cloned the project, run `yarn` to install all the dependencies.
 If you encounter trouble in the post-install phase involving `node-sass`, make sure you are building with VC++ 2013: `yarn config set msvs_version 2013`.
 
 #### Architecture :house:
-We have only one peer dependency on [lodash](https://lodash.com/) and we intend to keep it that way.<br />
-immutad●t is organized in namespaces (array, collection, lang, etc.), a lot like lodash, please try to respect this organization; if you are not sure where to put your code, ask for the right place in your issue or PR.
+immutad●t is managed as [monorepo](https://medium.com/netscape/the-case-for-monorepos-907c1361708a) using [lerna](https://lernajs.io/)
+
+The main package have no peer dependency and we intend to keep it that way.
```

Comment:
Most packages have no *peer* dependencies. Did you mean dependencies?

Reply:
Yes the best pratice when developping libraries is to put your dependencies as [peer dependencies](https://nodejs.org/uk/blog/npm/peer-dependencies/#the-solution-peer-dependencies) so you let host library or application choose the right version of your dependencies in case he also need them. Am i clear ?

Reply:
OK but when you say "it has no peer dependencies", I understand that it may have "classic" dependencies, while I guess you mean to say it has no dependencies at all, right? If you say "it has no dependencies" then I understand it has neither dependencies nor peer dependencies.